Tests/FluentAssertions.Specs/Types/TypeSelectorSpecs.cs

```diff
@@ -495,6 +496,42 @@ public void When_selecting_types_that_are_not_classes_it_should_return_the_corre
                 .And.Contain(typeof(NotOnlyClassesEnumeration));
         }
 
+        [Fact]
+        public void When_selecting_types_that_are_structs_it_should_return_the_correct_types()
```

Comment:
🤔 What about record structs?

Reply:
i will extend the tests for record structs and i think it should work fine with record structs as well.